test/scheduler/http.rb

```diff
@@ -0,0 +1,53 @@
+
+require 'benchmark'
+
+TOPICS = ["cats", "dogs", "pigs", "skeletons", "zombies", "ocelots", "villagers", "pillagers"]
+
+require 'net/http'
+require 'uri'
+require 'json'
+
+require_relative 'scheduler'
+
+def fetch_topics(topics)
+  responses = {}
+
+  topics.each do |topic|
+    Fiber.new(blocking: Fiber.current.blocking?) do
```

Comment:
How does this work? Does it register with the scheduler if blocking = false?

Reply:
It looks like it doesn't call `fiber` on the scheduler in this form, but isn't needed in this case since the scheduler doesn't do anything different.

Reply:
Yes, it's purely for convenience of showing the different behaviour.